tests/e2e/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "googlesitekit-e2e",
+  "description": "E2E testing infrastructure for Google Site Kit",
+  "main": "script.js",
+  "private": true,
+  "devDependencies": {
+    "expect-puppeteer": "^4.4.0",
+    "jest-puppeteer": "^4.4.0"
+  },
+  "scripts": {
+    "pretest:e2e": "./../../bin/local-env/env-check.sh && ./bin/test-docker-connectivity",
+    "test:e2e": "cross-env WP_BASE_URL=http://localhost:9002 ./script.js",
+    "test:e2e:interactive": "npm run test:e2e -- --puppeteer-interactive",
+    "test:e2e:watch": "npm run test:e2e -- --watch",
+    "test:e2e:watch:interactive": "npm run test:e2e -- --watch --puppeteer-interactive",
+    "env:start": "./../../bin/local-env/start.sh",
+    "env:stop": "./../../bin/local-env/stop.sh",
+    "env:reset-site": "./../../bin/local-env/install-wordpress.sh --reset-site"
```

Comment:
Maybe we should also move `local-env` files into the `tests/e2e` folder? We don't use it anywhere else, right?

Reply:
Ah yes good point, they are not used elsewhere.